doc/classes/NavigationMesh.xml

```diff
@@ -104,18 +105,23 @@
 		<member name="filter/low_hanging_obstacles" type="bool" setter="set_filter_low_hanging_obstacles" getter="get_filter_low_hanging_obstacles" default="false">
 		</member>
 		<member name="geometry/collision_mask" type="int" setter="set_collision_mask" getter="get_collision_mask">
+			The physics layers used to generate the [NavigationMesh].
 		</member>
 		<member name="geometry/parsed_geometry_type" type="int" setter="set_parsed_geometry_type" getter="get_parsed_geometry_type" default="0">
+			What kind of geomerty is used to generate the [NavigationMesh].
 		</member>
 		<member name="geometry/source_geometry_mode" type="int" setter="set_source_geometry_mode" getter="get_source_geometry_mode" default="0">
+			Which geometry is used to generate the [NavigationMesh].
 		</member>
 		<member name="geometry/source_group_name" type="StringName" setter="set_source_group_name" getter="get_source_group_name">
+			The name of the group that is used to generate the [NavigationMesh].
 		</member>
 		<member name="polygon/verts_per_poly" type="float" setter="set_verts_per_poly" getter="get_verts_per_poly" default="6.0">
```

Comment:
Count of vertices per each poly: a high number results in a lower resolution but much better performance.

Reply:
> The number of vertices to use per polygon. Higher values will improve performance at the cost of lower precision.

Reply:
Perfect!